gnomad/utils/annotations.py

```diff
@@ -1783,10 +1808,17 @@ def _agg_by_group(
                 lambda i: hl.agg.filter(ht.adj_array[i], agg_func(ann_expr[i], *args))
             )
         )
+        # Create final agg list by inserting or changing the "raw" group,
+        # representing all samples, in the adj_agg_list.
         raw_agg_expr = ann_expr.aggregate(lambda x: agg_func(x, *args))
-        # Create final agg list by inserting the "raw" group, representing all samples,
-        # into the adj_agg_list.
-        return adj_agg_expr[:1].append(raw_agg_expr).extend(adj_agg_expr[1:])
+        if group_membership_includes_raw_group:
+            adj_agg_expr[1] = raw_agg_expr
```

Comment:
I ran into error of this line while testing: 
`TypeError: 'ArrayStructExpression' object does not support item assignment`

Reply:
Sorry, some of my changes didn't get committed, I added them, so you shouldn't get this error anymore